src/screens/about/components/comparison.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import { Link } from "react-router";
+// Common
+import LOGO from "../../../../static/logotype.svg";
+import VLOGO from "victory-docs/static/logotype-hero.svg";
+
+class ComparisonChart extends React.Component {
+  render() {
+    return (
+      <div>
+        <h2>When should I use FormidableCharts versus Victory?</h2>
+        <p>We wanted to provide developers with multiple charting options for their projects. FormidableCharts is
+         quick and easy to implement with minimal muss and fuss around styling, theming, and composing components.
```

Comment:
Focus on the positives of each. Maybe something more like "focus on the data with pre-built, easy to use charts" vs . "Use Victory to create fully custom charts that integrate seamlessly with the look and feel of your project."